workspaces/sandbox/plugins/sandbox/src/hooks/useProductURLs.ts

```diff
@@ -50,7 +50,7 @@ export const productsURLMapping = (userData: SignupData | undefined) => {
     {
       id: Product.OPENSHIFT_CONSOLE,
       url: isProvisioned
-        ? `${userData?.consoleURL}/add/ns/${userData?.defaultUserNamespace}`
+        ? `${userData?.consoleURL}/k8s/cluster/projects/${userData?.defaultUserNamespace}`
```

Comment:
I think we could be a bit smarter here and check if consoleURL already has the trailing `/` and if so then don't add another one. That would help with avoiding `//` in the URL. We can address it in a separate PR though.